.github/workflows/bench.yml

```diff
@@ -40,3 +40,5 @@ jobs:
         alert-threshold: '130%'
         comment-on-alert: true
         alert-comment-cc-users: '@unbalancedparentheses'
+    - name: Clean benches
+      run: make clean
```

Comment:
Is this really necessary?

Reply:
I'm not sure what happens if I leave the files after running. I'm trying to avoid any possible re-compiling of files (and ending up with `file.cairo.json.json`). I'm guessing that the whole point of using make is to avoid that re-compiling, right?